graphql_compiler/compiler/emit_sql.py

```diff
@@ -998,17 +981,27 @@ def unfold(self):
 
         # 2. End the fold, collecting the folded subquery, the new from clause for the main
         # selectable, and the location of the folded outputs.
```

Comment:
Steps 2 and 3 of unfold should get a comment update. And possibly there should be an additional step between 3 and 4 for joining the fold subquery to the main selectable.